src/controllers/conversation.controller.ts

```diff
@@ -0,0 +1,22 @@
+import {
+    JsonController,
+    Get,
+    Authorized,
+    CurrentUser,
+  } from 'routing-controllers';
+  import { Service } from 'typedi';
+import { ConversationsService } from '@services/conversation.service';
+import { Conversation } from '@entities/conversation.entity';
+  
+  @JsonController('/conversations')
+  @Service()
+  export class ConversationController {
+    constructor(private readonly conversationsService: ConversationsService) {}
+  
+    @Authorized()
+    @Get()
+    async index(@CurrentUser() user :number): Promise<Conversation[]> {
+      return this.conversationsService.listConversationsByUser(user);
```

Comment:
The user param is the userId ? If that is the case I think the param can be renamed to userId in order to avoid confusion.

Also, maybe I am mistaken, but should not the @ CurrentUser return a user instead of an id ? If not, maybe we can think on renaming this to a more significant name like @CurrentUserId", what do you think?

Reply:
I agree with Pablo regarding to the user variable, maybe should be userId and I'm almost sure that this variable is a string, and definitely the name should be something with a little more context a very good practice is using descriptive name on your variables and functions in this way your code can be read for anyone (:

Reply:
Thanks, i will rename the parameter to userId and change the function name (index) to something better.

The userId is a number, as it was a number in the original code and I didn't change it to something like a UID yet.
I decided to return the id in @CurrentUser because I only needed the id and it could be obtained from the auth token, also because to obtain the complete user it would be necessary to access the database.

Also @CurrentUser is imported from routing-controllers and I can't rename it.